project_test.ipynb

```diff
@@ -90,14 +90,14 @@
     "       'Unnamed: 32', 'Unnamed: 33', 'Unnamed: 34', 'Unnamed: 35']].values[1:]\n",
     ")\n",
     "prelab_studentnum = prelab['Unnamed: 0'][1:].values\n",
-    "prelab_q = pd.concat(d, axis=1)\n",
+    "prelab_q = pd.concat(d, axis=1).astype('Int8')\n",
```

Comment:
why cast as int8? Save memory and time? Since the data is small, I think the benefits from casting as int8 might be minimal. what do you think

Reply:
It's a little neater since the default is float64 so our values would look like 1.0, 2.0, etc even when it should be whole numbers